swebench/harness/test_spec/test_spec.py

```diff
@@ -62,6 +64,16 @@ def install_repo_script(self):
 
     @property
     def base_image_key(self):
+        """
+        If base_docker_specs are present, the base image key is includes a hash of the specs.
```

Comment:
Since we configure php version in base dockerfile, we need to distinguish images with different versions. I hashed the `base_docker_specs` dict that configures this